<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhm</class>
 <widget class="QMainWindow" name="CIhm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>944</width>
    <height>877</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Supervision 2020</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>620</width>
      <height>610</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>510</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ressources/pers_vert.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="antenne">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../bin/ressources/lecteur_actif.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ressources/bt_ok.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ressources/vue_generale.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ressources/vue_lieu_1-2-3.jpg</pixmap>
      </property>
     </widget>
     <zorder>antenne</zorder>
     <zorder>label_4</zorder>
     <zorder>label_6</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Erreur</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>1241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Alarmes</string>
    </property>
    <widget class="QTextEdit" name="txtAlarme">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>1211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btClear">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Nettoyer</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Logiciel de Supervision 3.0 (2020)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbActivite">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ressources/activite_vert.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Erreur : le logiciel n'a pu se mettre à jour en fonction de la Base de données.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Erreur possible : La base de données est vide.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Si le problème persiste, contacter la section BTS IRIS qui a développé SuPer.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Erreur possible : La base de données est inaccessible.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Il est vivement conseillé de regarder attentivement l'aide fournie avec le logiciel.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBadge1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Badge 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBadge2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Badge 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBadge3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Badge 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBadge4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>430</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Badge 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBadge5">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>470</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Badge 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lImage2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>176</width>
      <height>144</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStop">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopper l'image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Caméras</string>
    </property>
   </widget>
   <widget class="QLabel" name="lImage">
    <property name="geometry">
     <rect>
      <x>755</x>
      <y>383</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reprendre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConfiguration">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>560</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Lancer SuPer_Configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDate">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>date/heure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SuPer : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Erreur possible : Le répertoire de travail n'est pas configuré.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNoZone">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>label_12</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_11</zorder>
   <zorder>label_7</zorder>
   <zorder>label_10</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>lbActivite</zorder>
   <zorder>tabWidget</zorder>
   <zorder>lbBadge1</zorder>
   <zorder>lbBadge2</zorder>
   <zorder>lbBadge3</zorder>
   <zorder>lbBadge4</zorder>
   <zorder>lbBadge5</zorder>
   <zorder>lImage2</zorder>
   <zorder>pbStop</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>lImage</zorder>
   <zorder>comboBox</zorder>
   <zorder>pbStart</zorder>
   <zorder>pbConfiguration</zorder>
   <zorder>lbDate</zorder>
   <zorder>label_2</zorder>
   <zorder>lNoZone</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
